modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. {rh-rhacm-first} ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the {rh-rhacm} hub cluster.
+
+For distributed units DUs, {rh-rhacm} supports single node OpenShift deployments. Single node OpenShift runs on a single bare-metal host. When the host is provisoned, the single node cluster acts as both supervisor and worker node. Shift Container Platform {product-title} on top of a single BMO For distributed units DUs, {rh-rhacm} supports single node OpenShift deployments. Single node OpenShift runs on a single bare-metal host. When the host is provisoned, the single node cluster acts as both supervisor and worker node.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+. Install {product-title} {product-verstion} on a hub cluster.
+. Install {rh-acm} and create the MultiClusterHub resource.
+. Create PersistentVolume custom resources (CR) for database and file system storage.
+. Install openshift-install CLI for the desired version of {product-title}.
+
+.Next Steps
+
+. Modify `HiveConfig` to enable the feature gate for Assisted Installer
++
+[source,terminal]
+----
+ oc patch hiveconfig hive --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"}}}'
+----
+
+. Modify the `provisioning` resource to allow the Bare Metal Operator to watch all namespaces
++
+[source,terminal]
+----
+ oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+----
+
+. Create an `AgentServiceConfig` custom resource
++
+[source,yaml]
+----
+ kind: AgentServiceConfig
+ metadata:
+   name: agent
+   namespace: assisted-installer
+ spec:
+   databaseStorage:
+     volumeName: <db_pv_name> <1>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <db_storage_size> <2>
+   filesystemStorage:
+     volumeName: <fs_pv-name> <3>
+     accessModes:
+     - ReadWriteOnce
+     resources:
+       requests:
+         storage: <fs_storage_size> <4>
+   osImages:
+     - openshiftVersion: <ocp_version> <5>
+       rootfs: <rootfs_url> <6>
+       url: <iso_url> <7>
+----
+<1> PersistentVolume name to use for database storage.
+<2> Volume size of the `db_pv_name` PersistentVolume.
+<3> PersistentVolume name to use for file system storage.
+<4> Volume size of the `fs_pv_name` PersistentVolume.
+<5> {product-title} version to boot the single node OpenShift clusters with.
```

Comment:
Probably shouldn't end the sentence with a preposition.

Reply:
Updated accordingly.